src/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,1367 @@
+            value="select"
+          />
+          <label
+            htmlFor="select"
+          >
+            Select
+          </label>
+        </div>
+        <div
+          className="inputForm"
+        >
+          <input
+            checked={false}
+            id="radios"
+            name="type"
+            onChange={[Function]}
+            type="radio"
+            value="radios"
+          />
+          <label
+            htmlFor="radios"
+          >
+            Radios
+          </label>
+        </div>
+      </div>
+      <div>
+        <label>
+          ADD MORE OPTIONS
+        </label>
+        <form
+          className="options-form"
+        >
+          <button
+            className="button-option"
+            onClick={[Function]}
+          >
+            ADD
+          </button>
+          <div>
+            <label>
+              Value
+            </label>
+            <input
+              className="input-option"
+              onChange={[Function]}
+              type="text"
+              value="1"
+            />
+            <input
+              className="button-option"
+              onClick={[Function]}
+              type="button"
+              value="DELETE"
+            />
+          </div>
+          <div>
+            <label>
+              Value
+            </label>
+            <input
+              className="input-option"
+              onChange={[Function]}
+              type="text"
+              value="2"
+            />
+            <input
+              className="button-option"
+              onClick={[Function]}
+              type="button"
+              value="DELETE"
+            />
+          </div>
+          <div>
+            <label>
+              Value
+            </label>
+            <input
+              className="input-option"
+              onChange={[Function]}
+              type="text"
+              value="3"
+            />
+            <input
+              className="button-option"
+              onClick={[Function]}
+              type="button"
+              value="DELETE"
+            />
+          </div>
+        </form>
+      </div>
+    </div>
+    <textarea
+      className="text-area"
+      onChange={[Function]}
+      value="{\\"label\\":\\"Label value\\",\\"type\\":\\"select\\",\\"options\\":[\\"1\\",\\"2\\",\\"3\\"]}"
+    />
+    <button
+      className="button"
+      onClick={[Function]}
+    >
+      Save
+    </button>
+    <button
+      className="button"
+      onClick={[Function]}
+    >
+      Cancel
+    </button>
+  </div>
+</div>
+`;
+
+exports[`Advanced tests of app  Selecting the type of radio buttons (select), add fields and input values then click cancel button 1`] = `
+<div>
+  <div
+    className="input-conntainer"
+  >
+    <input
+      className="input-field"
+      onClick={[Function]}
+      placeholder="Click here to for input text"
+      readOnly={true}
+      type="text"
+      value="Click here to for input text"
```

Comment:
value should be empty, so the placeholder is shown and not the text of the placeholder

Reply:
I will change this.